web-console/src/helpers/spec-conversion.ts

```diff
@@ -38,10 +38,10 @@ export function getSpecDatasourceName(spec: IngestionSpec): string {
 function convertFilter(filter: any): SqlExpression {
   switch (filter.type) {
     case 'selector':
-      return SqlRef.columnWithQuotes(filter.dimension).equal(filter.value);
+      return SqlRef.column(filter.dimension).equal(filter.value);
```

Comment:
Why don't we want quotes for filter and partition dimensions?

Reply:
In DQT 0.16.x the default behavior of `SqlRef.column` changed so `SqlRef.column` is now identical to `SqlRef.columnWithQuotes`. (This is why everything started adding quotes everywhere with relatively little code changes)

Reply:
Makes sense. `druid-query.ts` seems to be using `columnWithQuotes`. Should that just use `column` too?